sqlalchemy_cockroachdb/base.py

```diff
@@ -357,6 +358,24 @@ def get_foreign_keys_v1(self, conn, table_name, schema=None, **kw):
                 fkeys.append(fkey_d)
         return fkeys
 
+    @util.memoized_property
+    def _fk_regex_pattern(self):
+        # optionally quoted token
+        qtoken = r'(?:"[^"]+"|[A-Za-z0-9_\S]+?)'
```

Comment:
hm i'm reading this [SO answer](https://stackoverflow.com/questions/5028717/matching-unicode-characters-in-python-regular-expressions) and [these docs](https://docs.python.org/3/library/re.html).

i wonder if in python3 it would be sufficient to use:
```
qtoken = r'(?:"[^"]+"|[\w]+?)'
```

Reply:
and if we need python2 compatibility, then we could add the `re.UNICODE` flag when compiling the regex

Reply:
> i wonder if in python3 it would be sufficient

`qtoken = r'(?:"[^"]+"|[\w]+?)'` seems to be working fine.

> and if we need python2 compatibility

The change that raised the issue was part of the SQLA 2.0.21 release, and SQLA_2 is Python_3-only. It was not backported to SQLA_1.4 so nothing has changed for Python_2.